python/common/python/math/math_gamma.java

```diff
@@ -0,0 +1,242 @@
+package python.gammafunction;
```

Comment:
Should this gamma and the erf class be a separate package? I'm not sure I understand why it's done this way.